### PR TITLE
[RU] Enum improvements for Elixir 1.4

### PR DESCRIPTION
**Pull request by brain-geek**
One thing here though: please don't merge after review. As well as original #701 , this is intended to be merged after Elixir 1.4 release.

### CODE REVIEW

File: ru/lessons/basics/enum.md [hunk 1]
```diff
@@ -58,6 +58,15 @@ iex> Enum.chunk_by(["one", "two", "three", "four", "five", "six"], fn(x) -> Stri
 [["one", "two"], ["three"], ["four", "five"], ["six"]]
 ```
 
+### map_every
+
```
Comment (nscyclone) [inline ru/lessons/basics/enum.md]: This looks quite complicated, I'd rephrase the whole paragraph as following::

> Порой недостаточно просто разбить коллекцию на части. Если коллекция упорядочена должным образом, можно воспользоваться функцией `map_every/3`, позволяющей пройтись только по интересующим нас элементам.

File: ru/lessons/basics/enum.md [hunk 2]
```diff
@@ -135,11 +159,13 @@ iex> Enum.sort([%{:count => 4}, %{:count => 1}])
...
+Мы можем использовать `uniq_by/2` для удаления дубликатов из коллекций:
+ 
 ```elixir
-iex> Enum.uniq([1, 2, 2, 3, 3, 3, 4, 4, 4, 4])
-[1, 2, 3, 4]
+iex> Enum.uniq_by([1, 2, 3, 2, 1, 1, 1, 1, 1], fn x -> x end)
+[1, 2, 3]
 ```
+
+Ранее эта функция называлась `uniq/1`. Сейчас она переименована и при использовании будет выдавать предупреждение.
```
Comment (nscyclone) [inline ru/lessons/basics/enum.md]: I'd suggest to change the last sentence to make it more similar to the original one:
>Старый вариант помечен устаревшим в Elixir 1.4 и будет выдавать предупреждении при использовании.

**Comment by brain-geek** [comment]
>  You translate the newest updates so fast that I'm almost out of work. 😉

I still haven't got my hands to latest Mnesia changes. Feel free to take them. :)

**Comment by brain-geek** [comment]
Thanks for the feedback, @nscyclone . I've fixed both problems and now this PR is ready for merge after original #701 .

**Comment by ybur-yug** [comment]
@brain-geek can you change the base this points at to be my branch? I figure we shouldn't merge these to master so they dont deploy; since 1.4 isn't released yet it could cause some confusion.

**Comment by brain-geek** [comment]
@ybur-yug , I've made it simpler - pushed this commit to `improvement/elixir-1.4` branch, as it's already reviewed.
